Telcoautomation/README.md

```diff
@@ -0,0 +1,45 @@
+# Google Cloud Telcoautomation for PHP
+
+> Idiomatic PHP client for [Google Cloud Telcoautomation]().
```

Comment:
We are missing a documentation URL here. We need to either provide the URL, or remove the link.

Reply:
This URL should work: https://cloud.google.com/telecom-network-automation